lib/users/routes.js

```diff
@@ -71,16 +92,126 @@ TODO: inject content into form
 @function
 @name /users/edit/{{id}}
 */
-router.get('/edit/:id', (req, res) => {
-  // this render should populate the form
-  res.render('users/add', {
-    pageTitle: `${config.users.usersHome.title} :: Edit User`,
+router.get('/edit/:id',
+  loginMiddleware({
+    'role': 'superadmin',
+    'redirectTo': '/login',
+    'setReturnTo': '/users/add',
+  }),
+  (req, res, next) => {
+    req.session.editUserId = req.params.id; // eslint-disable-line no-param-reassign
```

Comment:
allTheCamelCase. `req.session.form.id` maybe. Something like that, so we can have a consistent place to throw sessions stuff for active form